crates/python/src/qpu/client.rs

```diff
@@ -210,6 +210,26 @@ impl PyQcsClient {
         })
     }
 
+    #[getter]
+    pub fn api_url(&self) -> String {
+        self.0.get_config().api_url().to_string()
```

Comment:
For consistency, can we `self.as_inner()` via the `rigetti-pyo3` `PyWrapper` trait?

Reply:
Didn't find a receiver method `.as_inner`, but `.as_ref` does the same 👍